.github/workflows/jest-balance.yml

```diff
@@ -1,12 +1,18 @@
 name: jest balancer
-on: [workflow_dispatch]
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * SUN'
 jobs:
   jest-balance:
     # Buckle up, this may take a while
     timeout-minutes: 60
     # Make sure this matches the runner that runs frontend tests
     runs-on: ubuntu-20.04
     steps:
+      - shell: bash
+        run: |
```

Comment:
@asottile-sentry I'm actually not sure if this is the best way, but we are trying to run this every two weeks on a sunday... Lmk if there is a better way

Reply:
if sunday isn't important could do 1,15 or whatever

Reply:
Yeah, I'll do that.